src/selectedtests/work_items/cli.py

```diff
@@ -7,49 +7,116 @@
     process_queued_test_mapping_work_items,
     process_queued_task_mapping_work_items,
 )
-from selectedtests.helpers import get_evg_api, get_mongo_wrapper, setup_logging
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.helpers import get_evg_api, setup_logging
+from selectedtests.evergreen_helper import get_evg_project
+from selectedtests.work_items.project_task_mapping_work_item import ProjectTaskMappingWorkItem
+from selectedtests.work_items.project_test_mapping_work_item import ProjectTestMappingWorkItem
+
+DEFAULT_WEEKS_BACK = 24
 
 
 @click.group()
 @click.option("--verbose", is_flag=True, default=False, help="Enable verbose logging.")
+@click.option("--mongo-uri", required=True, type=str, help="Mongo URI to connect to.")
 @click.pass_context
-def cli(ctx, verbose: str):
+def cli(ctx, verbose: str, mongo_uri: str):
     """Entry point for the cli interface. It sets up the evg api instance and logging."""
     ctx.ensure_object(dict)
-    ctx.obj["mongo"] = get_mongo_wrapper()
+    ctx.obj["mongo"] = MongoWrapper.connect(mongo_uri)
     ctx.obj["evg_api"] = get_evg_api()
 
     setup_logging(verbose)
 
 
 @cli.command()
+@click.option("--project", type=str, required=True, help="Evergreen project to add.")
+@click.option(
+    "--src-regex", type=str, required=True, help="Regular expression for project source files."
+)
+@click.pass_context
+def create_test_mapping(ctx, project: str, src_regex: str):
+    """
+    Add a project to the queue to be tracked for test mappings.
+
+    This command is meant for testing purposes only. Adding new projects to the task mappings
+    should normally be done via the REST API.
+    \f
+    :param ctx: Command Context.
+    :param project: Evergreen project to add to queue.
+    :param src_regex: Regular expression for project source files.
+    """
+    evergreen_project = get_evg_project(ctx.obj["evg_api"], project)
+    if not evergreen_project:
+        raise ValueError("Evergreen project not found")
+
+    work_item = ProjectTestMappingWorkItem.new_test_mappings(project, src_regex)
+    work_item.insert(ctx.obj["mongo"].test_mappings_queue())
+
+
+@cli.command()
+@click.option(
```

Comment:
I added this for testing purposes. When actually running against a project, the default of 6 months takes a long time to run. For running locally, I'd rather test 1 week.